From 07c3ad402a25454067e7bdf92a57fe77f8ca9922 Mon Sep 17 00:00:00 2001
From: vdanix <vishwanathx.dani@intel.com>
Date: Mon, 29 Nov 2021 16:05:13 +0530
Subject: [PATCH] Revert "Include VHAL for camera"

Revert the commit which enable

Legacy hal camera.We are facing
failures in VtsHalCameraProviderV2_4TargetTest

This reverts commit 52402f3f9feb24ff5ab7fc7cc4941c0d9f15eda7.

Tracked-On: OAM-100203

diff --git a/groups/camera-ext/ext-camera-only/product.mk b/groups/camera-ext/ext-camera-only/product.mk
index a39c34f..815464c 100644
--- a/groups/camera-ext/ext-camera-only/product.mk
+++ b/groups/camera-ext/ext-camera-only/product.mk
@@ -5,22 +5,9 @@ PRODUCT_COPY_FILES += \
 
 # External camera service
 PRODUCT_PACKAGES += android.hardware.camera.provider@2.4-external-service \
-                    android.hardware.camera.provider@2.4-service_64 \
                     android.hardware.camera.provider@2.4-impl
-#VHAL camera
-PRODUCT_PACKAGES += camera.$(TARGET_BOARD_PLATFORM) \
-                    camera.$(TARGET_BOARD_PLATFORM).jpeg
 
-PRODUCT_PROPERTY_OVERRIDES += ro.vendor.remote.sf.fake_camera ="both" \
-                              ro.vendor.camera.in_frame_format.h264=false \
-                              ro.vendor.camera.in_frame_format.i420=true \
-                              ro.vendor.camera.decode.vaapi=false \
-                              ro.vendor.remote.sf.back_camera_hal= \
-                              ro.vendor.remote.sf.front_camera_hal= \
-                              ro.vendor.camera.transference="VSOCK" \
-                              vendor.camera.external="VHAL"
-#removing not required apps
 # Only include test apps in eng or userdebug builds.
-#PRODUCT_PACKAGES_DEBUG += TestingCamera
+PRODUCT_PACKAGES_DEBUG += TestingCamera
 
 PRODUCT_PACKAGES += MultiCameraApp
diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index eed535a..5d28424 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -122,7 +122,6 @@
         <interface>
             <name>ICameraProvider</name>
             <instance>external/0</instance>
-            <instance>legacy/0</instance>
         </interface>
     </hal>
     <hal format="hidl">
diff --git a/groups/device-specific/caas_cfc/manifest.xml b/groups/device-specific/caas_cfc/manifest.xml
index 5a41ab8..499945f 100644
--- a/groups/device-specific/caas_cfc/manifest.xml
+++ b/groups/device-specific/caas_cfc/manifest.xml
@@ -149,7 +149,6 @@
         <interface>
             <name>ICameraProvider</name>
             <instance>external/0</instance>
-            <instance>legacy/0</instance>
         </interface>
     </hal>
     <hal format="hidl">
-- 
2.34.1

